prow/getting_started_deploy.md

```diff
@@ -148,8 +152,8 @@ kubectl apply -f config/prow/cluster/starter.yaml
 
 After a moment, the cluster components will be running.
 
-```console
-$ kubectl get deployments
```

Comment:
Removing the `$`s makes these less readable IMO.

Reply:
Should I add it to all of them now?  It was inconsistent :)

Reply:
Yes that would be great. I think it is helpful to distinguish the command that needs to be run from the command's output, but consistency is definitely most important.
Please also squash your commits.